tests/Domain/FundRepositoryTest.php

```diff
@@ -95,6 +103,17 @@ public function testPullForCampaignAllNew(): void
         $campaign->setSalesforceId('sfFakeId987');
 
         $repo->pullForCampaign($campaign);
+
+        $this->assertInstanceOf(CampaignFunding::class, $campaignFunding);
+        $this->assertInstanceOf(\DateTime::class, $campaignFunding->getCreatedDate());
+        $this->assertSame('1500', $campaignFunding->getAmount());
```

Comment:
I just checked the mutation report, this one is missing:

```suggestion
        $this->assertSame('1500', $campaignFunding->getAmount());
        $this->assertSame('GBP', $campaignFunding->getCurrencyCode());
```